vagrant/provisioning/roles/upgrade-solr/tasks/main.yml

```diff
@@ -94,10 +94,10 @@
         name: arkcase
         state: started
       when: _result.status == 200
-  when: solr_data_formated < space_available_formated
+  when: solr_data_formated > space_available_formated
 
```

Comment:
Let's say solr_data_formated is 5, and space_available_formated is 10.  Do we need 2x space_available as solr, or 1x?   If we make only one copy, then 1x is fine.  So in this example, 5 < 10 and by the new logic, we wouldn't proceed.  Isn't the old logic right?  Where we would proceed in this example?

Reply:
@david-oc-miller Please check this latest changes.